ida-ws/src/main/java/upb/ida/logging/LoggerProvider.java

```diff
@@ -0,0 +1,72 @@
+package upb.ida.logging;
+
+/**
+ * Class to expose logger provider methods for Aspect Logger in IDA
+ * 
+ * @author Ayaz
+ *
+ */
+
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.stereotype.Component;
+
+import upb.ida.util.FileUtil;
+
+@Component
+public class LoggerProvider {
+	
+	@Autowired
+	private FileUtil fileUtil;
+	
+	
+	/**
+	 * Method to get the logger instance for RequestResponseLogger from a log4j.properties file
+	 * 
+	 * @return - RequestResponseLogger
+	 */
+	@Bean
+	@Scope(value = "singleton", proxyMode = ScopedProxyMode.TARGET_CLASS)
+	@Qualifier("requestResponseLogger")
+	public Logger getRequestResponseLogger() 
+	{
+		String log4jConfigFile = fileUtil.fetchSysFilePath("log4j.properties");
```

Comment:
Put the literal "log4j.properties" as a constant  in https://github.com/dice-group/ida/blob/master/ida-ws/src/main/java/upb/ida/constant/IDALiteral.java#L28